source/use-cases/sensitive-data-encryption.txt

```diff
@@ -57,12 +57,122 @@ cubilia Curae; Nunc non interdum purus, ultricies laoreet tortor.
+    | Blood Type                  | Random              |  String                                      |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Medical Records             | Random              |  Array                                       |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Insurance: Policy Number    | Deterministic       |  Int (embedded inside `insurance` object)    |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Insurance: Provider         | Non-Encrypted       |  String (embedded inside `insurance` object) |
+    +-----------------------------+---------------------+----------------------------------------------+
+
+To automatically encrypt and decrypt create a JSON Schema using the information in the data model and pass the schema as a JSON object to the secure client to specify the encrypted field properties.
+In the JSON, you can specify `encryptMetadata <https://docs.mongodb.com/manual/reference/security-client-side-automatic-json-schema/#encryptmetadata-schema-keyword>`_ that is passed down to the fields under the ``properties``
+nested object. 
+
+.. code::
+
+        {
+            "bsonType" : "object",
+            "encryptMetadata" : {
+                "keyId" : // copy and paste your keyID generated here
+            },
+            "properties": {
+                // copy and paste your fields here 
+            }
+        }
+
+Create JSON objects for each field and append them to the `properties` map.
+
+.. code::
+
+    // Deterministic encryption supports querying and is equally as secure as random encryption for high cardinality (very unique) fields such as SSN. A privileged user in the MedcoMD Medical Care Management System might want to query on SSN to verify a patient's identity. One thing to note is that you do not have to include keyId if you specified it in the top-level encryptMetadata field. Though you could include it if you want different users with specific keys to be able to access a specific subset of the encrypted data.
+
+    "ssn": {
+        "encrypt": {
+            "bsonType": "int",
+            "keyId": <keyID>,
+            "algorithm": "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic"
+        }
+    }
+
+
+.. code:: javascript
+
+    // Random encryption does not support filtering and is best used on fields with low cardinality (very few unique fields) to prevent against frequency analysis attacks. Since there are only 8 blood types, this field is randomly encrypted. To decrypt this data, a privileged user would have to query on another field via a secure collection, and this data would be returned unencrypted.
+
+    "bloodType": {
+        "encrypt": {
+            "bsonType": "string",
+            "keyId": <keyId>,
```

Comment:
Nitpick: the other instances use `<keyID>`, would be good to standardize.

Reply:
Most actually use <keyId>, fair point on standardizing. I've switched all to "keyId" since that is what the property is as well. I think that is the most clear